### PR TITLE
Cxa thread atexit shim

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: src/cxaThreadAtExitShim.cxx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  using Dtor = void(*)(void*);
+
+}
+
+extern "C" {
+
+int __cxa_thread_atexit(__cxxabiv1::Dtor dtor, void* obj, void* dso_symbol) throw()
+{
+  // Do nothing
```
Comment (N-Dekker) [inline src/cxaThreadAtExitShim.cxx:33]: Should it not `return` anything?

Comment (thewtex) [reply src/cxaThreadAtExitShim.cxx:33]: We could have it return an `int`.

Comment (N-Dekker) [reply src/cxaThreadAtExitShim.cxx:33]: OK, but then it should have a `return` statement, right?

Comment (thewtex) [reply src/cxaThreadAtExitShim.cxx:33]: Yes, it now returns -1, which seems to be what is used when the function does not run as expected.

Comment (N-Dekker) [reply src/cxaThreadAtExitShim.cxx:33]: Thanks for addressing my comment 👍

**Pull request merged**